ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetObjectVersionsView.java

```diff
@@ -0,0 +1,49 @@
+package com.spectralogic.ds3cli.views.cli;
+
+import com.bethecoder.ascii_table.ASCIITable;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import com.spectralogic.ds3cli.models.GetObjectVersionsResult;
+import com.spectralogic.ds3client.models.S3Object;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.spectralogic.ds3cli.util.Constants.DATE_FORMAT;
+import static com.spectralogic.ds3cli.util.Guard.nullGuard;
+import static com.spectralogic.ds3cli.util.Guard.nullGuardFromDate;
+
+public class GetObjectVersionsView extends TableView<GetObjectVersionsResult> {
+
+    private Iterable<S3Object> versions;
```

Comment:
missing `final`?

Reply:
This cannot be final since the version is not known until `render` is called after the constructor has completed.